.circleci/config.yml

```
@@ -73,7 +73,7 @@ jobs:
   end-to-end-test:
       docker:
         - image: circleci/node:8-browsers
-        - image: grafana/grafana:master
+        - image: grafana/grafana-dev:$CIRCLE_SHA1
```

```suggestion
        - image: grafana/grafana-dev:master-$CIRCLE_SHA1
```

See: https://hub.docker.com/r/grafana/grafana-dev/tags

---

Good catch! I'll fix